test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Import/StartupForImportBasicSearchTestProvider.cs

```diff
@@ -0,0 +1,26 @@
+﻿// -------------------------------------------------------------------------------------------------
```

Comment:
Is it possible for the test fixtures to use one common startup class instead of having 10 duplicate files?

Reply:
we do not need seperate db for each test, merged to single provider.